src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -1,23 +1,24 @@
 package edu.oregonstate.cs361.battleship;
 
 import java.util.ArrayList;
+import java.util.Random;
 
 /**
  * Created by michaelhilton on 1/4/17.
  */
 public class BattleshipModel {
 
-    private Ship aircraftCarrier = new Ship("AircraftCarrier",5, new Coordinate(0,0),new Coordinate(0,0));
-    private StealthShip battleship = new StealthShip("Battleship",4, new Coordinate(0,0),new Coordinate(0,0));
-    private StealthShip submarine = new StealthShip("Submarine",3, new Coordinate(0,0),new Coordinate(0,0));
+    private MilitaryShip aircraftCarrier = new MilitaryShip("AircraftCarrier", 5, new Coordinate(0, 0), new Coordinate(0, 0), false);
```

Comment:
So it makes everything a bit cleaner and It will allow overriding a function in Ship that deals with the scan. Civilian will be doing something simular. Not fully implemented with this pull request but I now have the code ready and will put it up tommorrow morning. I asked the TA about it and he said it is much better. So you can wait to accept this if you'd like